File: Java version/app/src/main/java/it/unical/mat/embasp/specializations/clingo/ClingoAnswerSets.java
package it.unical.mat.embasp.specializations.clingo;

import it.unical.mat.embasp.languages.asp.AnswerSets;
import it.unical.mat.parsers.asp.ASPSolversParser;

/** represents Clingo's answersets */

public class ClingoAnswerSets extends AnswerSets {

	public ClingoAnswerSets(final String answersets) {
		super(answersets);
	}

	public ClingoAnswerSets(final String out, final String err) {
		super(out, err);
	}

	@Override
	protected void parse() {
		ASPSolversParser.parseClingo(this, output, true);
	}
}

File: CSharp version/it/unical/mat/embasp/specializations/clingo/ClingoAnswerSets.cs
using it.unical.mat.parsers.asp;

namespace it.unical.mat.embasp.specializations.clingo
{
    using AnswerSets = it.unical.mat.embasp.languages.asp.AnswerSets;

    public class ClingoAnswerSets : AnswerSets
    {
        public ClingoAnswerSets(string answersets) : base(answersets) { }

        public ClingoAnswerSets(string @out, string err) : base(@out, err) { }

        protected internal override void Parse()
        {
            ASPSolversParser.ParseClingo(this, output, true);
        }
    }
}